docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_label: 'About GX Cloud'
+title: 'About GX Cloud'
+description: Learn more about the features and benefits of GX Cloud.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+GX Cloud builds on the innovative features of GX open source software (OSS). Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
+
+Here are just a few of the benefits you'll get with GX Cloud: 
+
+- It's web-based and browser- and platform-independent.
+
+- A hosted environment and intuitive interface makes it faster to set up and run your data Validations.
+
+- Features such as Validation run histories, Data Asset metrics, and more.
+
+- No more need to create, run, and maintain complex code for your data queries.
+
+## Workflow optimization
+
+GX Cloud helps you get started faster and simplifies your data Validation workflows. Here's how:
+
+- You can identify your Data Source, connect to a Data Asset, create Expectations, and then run Validations without running a single line of Python code. 
+
+- You can reuse or modify your Expectations as your business requirements change. No need to create new Expectations every time you need to validate your data.
+
+- You can execute Validations with the click of a button and get your results immediately.
+
+- You can share Validation data quickly and make it accessible to all users regardless of their technical knowledge.
+
+## GX Cloud workflow
+
+The GX Cloud workflow is a sequence of tasks you complete to perform Data Validations. 
+
+You connect GX Cloud to a Data Asset stored on your Data Source, you define and create an Expectation, and then you run a Validation on the data defined in the Expectation. Knowing that your data meets your Expectations helps you ensure your data is accurate and reliable and alerts you to issues before they become problematic.
+
+The following table lists the tasks that you complete in a typical GX Cloud workflow. A brief description of the task and links to the relevant topics are provided.
+
+| Task                                     | Description                                                        | Related topic               |
+|------------------------------------------|--------------------------------------------------------------------|-----------------------------|
+| Set up GX Cloud                          | Set up your environment to get the most out of GX Cloud. Do it once, and you don't need to do it again.           | [Set up GX Cloud](/docs/cloud/set_up_gx_cloud)                         | 
+| Connect to a Data Asset on a Data Source | The Data Asset contains the data you want to examine and Validate. | [Create a Data Asset](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset)                           | 
```

Comment:
nit: can we change "Validate" to "validate"?

Reply:
@HaebichanGX done! Thanks.